skt/templates/partials/test_failure.j2

```diff
@@ -11,10 +11,12 @@ One or more hardware tests failed:
   {% for test_result in job.test_results %}
     {% for failed_task in test_result.failed_tasks %}
     Test name: {{ failed_task.name }}
+      {% if not report_omit_hardware_test_logs %}
```

Comment:
I don't like different parts of the report code communicating via global variables, but if there's no other option, so be it.

Reply:
That's the only option I've found so far. Let me know if you think of something else. ;)